### ISSUE TITLE
xarray version 0.16.0 breaks xmitgcm.open_mdsdataset

### ISSUE THREAD

**Issue by cspencerjones** [MITgcm/xmitgcm#227]
I am using the latest release of `xmitgcm`, 0.4.1. 

When `xarray` is updated past version 0.16.0, `xmitgcm.open_mdsdataset` fails with the error:

```
~/.conda/envs/pangeo2/lib/python3.7/site-packages/xmitgcm/mds_store.py in open_mdsdataset(data_dir, grid_dir, iters, prefix, read_grid, delta_t, ref_date, calendar, geometry, grid_vars_to_coords, swap_dims, endian, chunks, ignore_unknown_vars, default_dtype, nx, ny, nz, llc_method, extra_metadata)
    215                                         **kwargs))
    216                 # apply chunking
--> 217                 ds = xr.auto_combine(datasets)
    218                 if swap_dims:
    219                     ds = _swap_dimensions(ds, geometry)

AttributeError: module 'xarray' has no attribute 'auto_combine'
```

Previous deprecation warnings indicate that the `auto_combine` method has been replaced by `combine_by_coords`, so we should probably update `mds_store.py` to use that method instead.

**Comment by rabernat** [comment]
We just need to release xmitgcm in order to fix this! It has already been fixed on master:

https://github.com/MITgcm/xmitgcm/blob/3ab1fdba44c9c2a216a863df604039eee6132dca/xmitgcm/mds_store.py#L236-L241

**Comment by ifenty** [comment]
@rabernat same issue with my production codes, please send updated xmitgcm to conda

**Comment by rabernat** [comment]
Hi everyone! We will try to get a release out asap! 🙃

**Comment by rabernat** [comment]
This is fixed in the 0.5.0 releasel.
